src/components/PromptSignIn/PromptSignIn.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+import {Keyset, i18} from '../../i18n';
+import {Prompt, PromptProps} from '../Prompt/Prompt';
+
+export interface PromptSignInProps extends Partial<PromptProps> {
+    onClickSignIn?: React.MouseEventHandler;
+}
+
+/**
+ * Authentication Popup that appears when user action requires login
+ * @returns {JSX|null}
+ */
+export const PromptSignIn: React.FC<PromptSignInProps> = ({
```

Comment:
Are we really need this component in package ? Maybe our user will be create their wrappers in their applications ?

Reply:
Yes, because:

1. You said you wanted a built-in login popup in blog constructor, so that minimal code is required for implementation.
2. It ensures consistent implementation between projects, by keeping it DRY
3. It is very important for users from UX point of view to receive consistent message between apps

Reply:
Ok, I thought we export this wrapper outside because i don't see where `PromptSIgnIn` render in BlogPage
We can use only Prompt component in BlogPage, without this wrapper, because users will be customize actions and texts in propmt in their app, via props

Reply:
`PromptSignIn` implementation (ie. placements) is inside another repo. Here, it only exports itself as interface for consistency and DRY reasons. 

I chose this strategy to allow for a flexible implementation of the PromptSignIn placement, and also because the `onClick` handler that opens popup by changing `openTimestamp` cannot be implemented in this repo - since it could be anywhere in the code. 

You'll see implementation once this pr is merged, and I get package version number.

Reply:
Yes, we can implement `PromptSignIn` in different places in different app, but why , if we need it on `BlogPage` ? We immediately give users such a feature on the blog page, and they do not need to write an implementation in their application. We can also leave the option to export that component and implement it in any places if someone needs it. But I'm not sure that this will happen, because we don't have a library of common components, but a narrower library for a blog